### PR TITLE
Add Chinese (Taiwan) strings with change Chinese (China) strings

### PR DESCRIPTION
**Pull request by LNDDYL**
新增 _**中文 (台灣)**_ 字串，並變更 _**中文 (中國大陸)**_ 字串。

### CODE REVIEW

File: src/main/res/values-zh-rCN/strings.xml [hunk 1]
```diff
@@ -61,7 +61,7 @@
   <string name="flush_dnscache_no_root">在没有 ROOT 权限的情况下无法刷新 DNS 缓存</string>
   <string name="flushing">刷新中……</string>
   <string name="share">分享</string>
-  <string name="share_message">扫描二维码或者使用 NFC（Android Beam）</string>
+  <string name="share_message">扫描 QR 代码或者使用 NFC（Android Beam）</string>
```
Comment (madeye) [inline src/main/res/values-zh-rCN/strings.xml]: 二维码 is preferred here.

File: src/main/res/values-zh-rCN/strings.xml [hunk 2]
```diff
@@ -93,7 +93,7 @@
   <string name="gotcha">好的</string>
   <string name="copy_url">复制 URL</string>
   <string name="add_profile_dialog">为影梭添加此配置文件？</string>
-  <string name="add_profile_methods_scan_qr_code">扫描二维码</string>
+  <string name="add_profile_methods_scan_qr_code">扫描 QR 代码</string>
```
Comment (madeye) [inline src/main/res/values-zh-rCN/strings.xml]: Ditto.

**Comment by LNDDYL** [comment]
二維碼並不等於QR碼，QR碼只是二維碼下的分支。
Microsoft Language Portal: https://www.microsoft.com/Language/en-US/Search.aspx?sString=QR%20Code&langID=zh-cn

**Comment by Mygod** [comment]
Correct but in mainland people just assumes 2D codes are QR codes and calling them 2D codes causes confusion.

**Comment by LNDDYL** [comment]
把QR碼等同於二維碼是錯誤的，反之亦然。

**Comment by Mygod** [comment]
As I've said it's about following conventions, not about academic correctness. Calling them that is not wrong in any case and better at the same time which avoids ambiguity but it's just that few people in mainland really says that (and some people doesn't even know what's QR code).

**Comment by madeye** [comment]
Sorry, but if you want to have this pull request merged, you need to use the "wrong" string.

And of course, without your permission, I won't modify your changes.

**Comment by LNDDYL** [comment]
英文檔 裡面寫的是 QRCode ，而不是 2DCode 。
https://github.com/shadowsocks/shadowsocks-android/blob/master/src/main/res/values/strings.xml

**Comment by LNDDYL** [comment]
我是按照English原文翻譯的，翻譯不能亂譯。

**Comment by madeye** [comment]
https://translate.google.com/m/translate#en/zh-CN/QR%20code

**Comment by Mygod** [comment]
Trust a native speaker, not Microsoft. Their translation for Windows 10 is so poor that it makes me want to puke. And I can tell you 二维码 in Chinese (simplified) translates to QR code in English (despite its literal meaning) and vice versa.

**Comment by Mygod** [comment]
P.S. Mmpfft. Even "QR 码" is a better translation. Microsoft really needs a better translator. Just saying.

**Comment by AnyOfYou** [comment]
Reminds me of I used to insisted use KiB instead of KB in my App.

**Comment by LNDDYL** [comment]
@Mygod @madeye @AnyOfYou
我們各退一步，你們不提2D二維，我也不講QR，直接省作「碼」(code)，如何？

**Comment by LNDDYL** [comment]
> 二維條碼的種類很多，不同的機構開發出的二維條碼具有不同的結構以及編寫、讀取方法。常見的二維條碼有：
PDF417碼
QR碼
漢信碼
顏色條碼
EZ碼
Aztec Code
QuickMark
Data matrix

**Comment by wongsyrone** [comment]
I suggest keeping zh-CN as is.

**Comment by LNDDYL** [comment]
@madeye 麻煩你回覆一下。

**Comment by Mygod** [comment]
I insist that we should use 二维码.

**Comment by LNDDYL** [comment]
QR碼是2D碼中的一個，2D碼裡面包含QR碼，2D碼跟QR碼不是對等關係。

**Comment by Mygod** [comment]
See https://github.com/shadowsocks/shadowsocks-android/pull/612#issuecomment-197836680.

I have no idea how I can make this more clear to you.

**Comment by madeye** [comment]
As mentioned above, you have to make a decision if you'd like to get this pull request merged.

Another option is to open a new pull request with zh-TW translation only, I am glad to merge that patch.

**Comment by LNDDYL** [comment]
@madeye 把英文區的QRCode改成BarCode，可以嗎？

**Comment by madeye** [comment]
It'd be QR code, not barcode. https://shadowsocks.org/en/config/quick-guide.html

**Comment by LNDDYL** [comment]
@madeye 可是，QR code翻譯成中文就是QR碼啊，不是二維碼。

**Comment by LNDDYL** [comment]
稍後我會Pull上zh-TW譯文。

**Comment by LNDDYL** [comment]
@madeye https://github.com/shadowsocks/shadowsocks-android/pull/615

**Comment by ShikiSuen** [comment]
就是因为你们这样得过且过，简体中文资讯电子术语体系才会变得像今天这样一塌糊涂。哦，习惯的错误就可以堂而皇之地以「习惯、方便」为由流传下去？笑话。

**Comment by ShikiSuen** [comment]
@madeye 诉诸谷歌不见得就是诉诸正确，至少在这个词的翻译方面是错误的。谷歌新增粤语翻译引擎时都自认错误百出、还欢迎香港乡民去揪错呢。

**Comment by madeye** [comment]
Locked as off-topic.

**Pull request closed without merging**